sillsdev/web-languageforge
# fix deprecated gha apis

Fixes #1550 

## Description

GHA recently deprecated some apis and this PR migrates to the newer ones.

### Type of Change

- Bug fix (non-breaking change which fixes an issue)

## Screenshots

N/A

## Checklist


## How to test

- Double check workflow runs on a pull request and ensure there are no deprecation warnings.
- Double check workflow runs on a a merge to `develop` and ensure there are no deprecation warnings.

## qa.languageforge.org testing

N/A
